cfpb/hmda-frontend
# DB 2018 and 2019 test updates

Closes #519 

Large diff due to adding conditional block around test cases execution.

## Testing
- Set `CYPRESS_HOST` to `dev-beta` url => 2018 should run, 2019 should not
- Set `CYPRESS_HOST` to `prod-default` url => Both 2018 and 2019 db tests should run
- Set `CYPRESS_HOST` to `dev-default` url => Both 2018 and 2019 db tests should run
